src/main/java/duke/models/tasks/TaskManager.java

```diff
@@ -60,18 +60,30 @@ public TaskManager() {
         return tasksWithThisDescription;
     }
 
-
     /**
      * Adds a Task to the list.
      *
-     * @param task The Task to be added to the list.
+     * @param taskToBeAdded contains the Task to be added to the list.
+     * @throws DukeException if the task is not added successfully.
      */
-    public void addTask(Task task) {
-        if (task.getId() == 0) {
+    public void addTask(Task taskToBeAdded) throws DukeException {
+        if (taskToBeAdded.getDescription().length() < 4) {
```

Comment:
I'm wondering if it would be better to require just strings that aren't <= 0; some tasks can be shorter than 4 letters, such as `run` or `eat`.

Reply:
I will change to 3. I think the description with less than 3 characters doesn't really make sense.